lorax.spec

```diff
@@ -73,6 +73,20 @@ It also includes livemedia-creator which is used to create bootable livemedia,
 including live isos and disk images. It can use libvirtd for the install, or
 Anaconda's image install feature.
 
+%package composer
+Summary: Lorax Image Composer API Server
+# From EPEL
+Requires: python2-pytoml
+Requires: python-semantic_version
+Requires: libgit2
+Requires: libgit2-glib
+# From Distribution
+Requires: python-flask
+Requires: python-gevent
+
+%description composer
+lorax-composer provides a REST API for building images using lorax.
+
```

Comment:
According to the fedora wiki we need some scriptlets for the service, too. I think:

```
%post composer
%systemd_post lorax-composer.service

%preun composer
%systemd_preun lorax-composer.service

%postun composer
%systemd_postun_with_restart lorax-composer.service
```

plus

```
%{?systemd_requires}
BuildRequires: systemd
```

for the script Requires and the macro BuildRequires

Reply:
Thanks, for some reason I was thinking those would enable it, but it looks like they're using the presets files to make that decision.